# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../sysvinit/sulogin_initscr.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Rediffed fro 2.86 (indentation change).

  - Rene Rebe <reneÂ@exactcode.de>

--- sysvinit-2.86/src/init.c.orig	2004-07-30 14:16:20.000000000 +0200
+++ sysvinit-2.86/src/init.c	2004-12-11 11:36:04.901749888 +0100
@@ -93,7 +93,7 @@
 	"~~",
 	"S",
 	3,
-	"/sbin/sulogin",
+	"/etc/initscript - - - /sbin/sulogin",
 	NULL,
 	NULL
 };
